### DISCUSSION TITLE
User Authentication using API

### DISCUSSION THREAD

**Discussion by Kanishka-Deshan** [snipe/snipe-it#15326] [General]
First of all I'm not an expert of coding but know few. I have an issue that I'm developing a web app where the user authentication should be done with snipe-it user table. Simply in my app login page username and password should authenticate by snipe user table. Is there any way to full fill this. I do not want create another user table in my app because there are more than 500 users in that table. Please help me to overcome this.

**Comment by snipe** [comment]
I don't think that's going to work though. We do not store passwords in plain text, they are in a one-way hash, so your other app would not be able to authenticate with it using the data in Snipe-IT.

**Reply by Kanishka-Deshan** [reply]
Thank You so much @snipe ! Really appreciate your advice.
